common/actions/config.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+export const CONFIG_LANGUAGE_CHANGE = 'CONFIG_LANGUAGE_CHANGE';
+export const CONFIG_LANGUAGE_DROPDOWN_TOGGLE = 'CONFIG_LANGUAGE_DROPDOWN_TOGGLE';
+export const CONFIG_NODE_CHANGE = 'CONFIG_NODE_CHANGE';
+export const CONFIG_NODE_DROPDOWN_TOGGLE = 'CONFIG_NODE_DROPDOWN_TOGGLE';
+
+
+export const CHANGE_LANGUAGE = (index: number) => Object({
```

Comment:
We want to change to language FOO, not language 1. Also fuck redux boilerplate :(